tests/test_booster/test_plugin/test_gemini_plugin.py

```diff
@@ -86,7 +86,8 @@ def check_gemini_plugin(subset: str, init_method: str = 'none', early_stop: bool
                 'transformers_t5_encoder_model',    # does not support apex rmsnorm
                 'transformers_chatglm',
                 'transformers_sam',
-                'transformers_vit'
+                'transformers_vit',
+                'transformers_gpt_double_heads'
```

Comment:
What's wrong with this?

Reply:
The test failed to run this model with gemini and raise a error that "synchronization of gradients doesn't exit properly",